src/graphics/device.js

```diff
@@ -27,7 +27,11 @@ pc.extend(pc, function () {
     };
 
     var _createContext = function (canvas, options) {
+        // #ifdef WEBGL2
```

Comment:
But that means it is not tested well. Changing it earlier might highlight some problems.